src/Util/PHP/Template/TestCaseClass.tpl.dist

```diff
@@ -26,6 +26,10 @@ if ($composerAutoload) {
 
 function __phpunit_run_isolated_test()
 {
+    // If testcase open stdout and close it, the output will be lost.
+    // So we open it manually.
+    $__stdout = fopen('php://stdout', 'w');
```

Comment:
1. I see a confusion between this handler and `STDOUT`, especially as both are used in very same method

2. I saw issues that raised from doing that things - opening a file creates a file handler, which are not free and unlimited - vide `FD_SETSIZE`, please close it as soon as it's not needed anymore

3. btw, it's only variable here with leading `__`, what for?

Reply:
1. As global, `STDOUT` may be closed by a test case. When `STDOUT` closed, the test result in subprocess will be lost.
2. I will rename this variable.
3. If there is a bug, and we can fix it, why not fix it quickly?

Reply:
1. still confusing, you just gave next reason to use `$stdout` over `STDOUT` in `stream_get_contents(STDOUT)`
2. :+1:
3. As Sebastian and me already answered, to ensure this is a bug and doesn't bring new issues